vagrant/provisioning/roles/arkcase-app/templates/arkcase-portal-prc-server.yaml

```diff
@@ -2,7 +2,16 @@ portal-prc:
   arkcaseUrl: "https://{{ external_host }}/arkcase"
   userId: "{{ arkcase_admin_user }}@{{ ldap_user_domain }}"
   password: "{{ arkcase_admin_password }}"
+  
+  s3:
+    url: "{{ file_management_s3_url }}"
+    region: "{{ file_management_s3_region }}"
+    accessKey: "{{ file_management_s3_access_key }}"
+    secretKey: "{{ file_management_s3_secret_key }}"
+    bucketName: "{{ prc_dockets_s3_bucket_name }}"
+  
 arkcase.rca.solr.configuration:
   host: "{{ solr_host }}"
-  port: {{ solr_port }}
+  port: {{ solr_port | default(8983) }}
```

Comment:
We found the default port should be 443 right?

Reply:
yes, I looked at other places and it is 443

Reply:
My bad, I made that change but didn't stage it.